doc/manage-duniverse.mld

```diff
@@ -0,0 +1,213 @@
+}
+
+If comes however with some caveats:
+
+{ul
+{- Everyone might end up with different packages in their duniverse, thus low
+   reproducability}
+{- Not self-contained}
+}
+
+{3 Adoption}
+
+Using this workflow is very simple
+
+{[
+$ opam monorepo lock
+$ opam monorepo pull
+]}
+
+This will lock the versions of packages into a lock file and will download the
+sources into the [duniverse] folder. Thus a subsequent [dune build] will also
+build the dependencies.
+
+To make sure the files don't get committed by accident, it is useful to add the
+following excludes to the [.gitignore] file:
+
+{[
+*.opam.locked
+duniverse/
+]}
+
+{2:lockfile-in-git Lockfile in Git}
+
+Another approach is very similar to the regular opam workflow, but in addition
+to having the [.opam] file committed to the repository, also the lockfile that
+[opam monorepo lock] generates is added.
+
+Advantages include:
+
+{ul
+{- The project repository only grows by one file, thus easy to adopt}
+{- The packages to be installed are reproducible between different machines
+   and contributors}
+{- Easier management of dependencies over dependencies installed in opam
+   switches}
+{- Having the dependencies built by Dune makes cross-compilation easier}
+}
+
+Disadvantages might be
+
+{ul
+{- Everyone who wants to build the dependencies with Dune needs to have
+   [opam-monorepo] installed to pull the dependencies first}
+{- Not self-contained}
+}
+
+{3 Adoption}
+
+To adopt this workflow, the initial lockfiles needs to be created:
+
+{[
+$ opam monorepo lock
+]}
+
+This creates one [.opam.locked] file for each [.opam] file, thus these need to
+be added to the repository
+
+{[
+$ git add *.opam.locked
+]}
+
+As [opam monorepo pull] creates a [duniverse] folder with all the sources of
+the vendored dependencies, it should be excluded by an entry in the
+[.gitignore] file:
+
+{[
+duniverse/
+]}
+
+{3 Dealing with conflicts}
+
+Since multiple contributors can run [opam monorepo lock] which will produce a
+different lock file depending on the state of the used OPAM repositories. This
+might lead to merge conflicts of the lockfiles when merging.
+
+As such instead of manually trying to resolve the conflict, it is suggested to
+replace the the conflicting lock file with a new lock file generated by [opam
+monorepo lock] - as long as the original [.opam] file has the right version
+ranges specified the new lock file will produce a valid result for the merged
+project.
+
+{2:duniverse-in-git Duniverse in Git}
+
+The most extreme approach is to create a lockfile with [opam monorepo lock],
+download all the dependencies via [opam monorepo pull] and then commit both the
+lockfile as well as the pulled sources.
+
+This appraoch has some advantages:
+
+{ul
+{- Only the person that creates the lockfile and duniverse must have
+   [opam-monorepo] installed}
+{- The repository can be immediately used after cloning as everything is
+   self-contained}
```

Comment:
(maybe specify that you just need `dune` and `ocaml` installed - `opam`doesnt need to be installed nor configured)